caps/capabilities.go

```diff
@@ -95,6 +99,16 @@ func NewRepository() *Repository {
 	}
 }
 
+// AddBuiltInTypes adds all built-in types to the repository
+// If any of the additions fail the function panics.
+func (r *Repository) AddBuiltInTypes() {
```

Comment:
Feels like this shouldn't be part of the repository API. I suggest a top-level along the lines of:

    func LoadBuiltIn(r *Repository) {
            ...
    }

In the future we'll be moving these built-ins out of the binary altogether, and the repository API still won't care.